catalog/transformer/catalog-transformer-streaming-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -64,7 +64,7 @@
                 <list>
                 </list>
             </property>
-
+            <property name="context" ref="blueprintBundleContext"/>
```

Comment:
do not pass in the `bundleContext` as it can become stale.  It is best to use FrameworkUtils to get a fresh bundle context each time you need it.
https://github.com/codice/ddf/blob/master/catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java#L1401